.flowconfig

```diff
@@ -1,9 +1,10 @@
 [ignore]
-.*/lib/*
+lib/*
```

Comment:
well `lib/*` works 😛 
and it doesn't exclude `node_modules/*/lib` and that's the only thing we need